docs/root/intro/arch_overview/load_balancing/load_balancing.rst

```diff
@@ -0,0 +1,15 @@
+Load Balancing
+==============
+
+.. toctree::
+  :maxdepth: 2
+
+  overview
+  load_balancers
+  original_dst
+  overprovisioning
```

Comment:
From reading the docs start to finish, I found the introduction of overprovisioning before priority a bit out of place. Perhaps either move after priority or just merge into the priority page? Thoughts?

Reply:
I think maybe overprovisioning should come after both locality weights and priority since it applies to both in a similar way. How about changing 

```
  overprovisioning
  priority
  zone_aware
  panic_threshold
  locality_weight
```

into 
```
  priority
  locality_weight
  overprovisioning
  zone_aware
  panic_threshold
```

?